packages/rollup-contracts/contracts/SequencerBatchSubmitter.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+/* Internal Imports */
+import {DataTypes as dt} from "./DataTypes.sol";
+import {RollupMerkleUtils} from "./RollupMerkleUtils.sol";
+import {CanonicalTransactionChain} from "./CanonicalTransactionChain.sol";
+import {StateCommitmentChain} from "./StateCommitmentChain.sol";
+
+contract SequencerBatchSubmitter {
```

Comment:
Could you add some context comment as to what this is meant to do, who is meant to interact with it, etc.?